docs/versioned/spark/spark-writes.md

```diff
@@ -98,6 +98,11 @@ WHEN NOT MATCHED AND s.event_time > still_valid_threshold THEN INSERT (id, count
 
 Only one record in the source data can update any given row of the target table, or else an error will be thrown.
 
+!!! Note
+    By default Spark uses copy-on-write merge mode.
+    With spark-3.2 and onwards, iceberg supports merge-on-read mode.    
+    To use merge-on-read merge mode, need to set the table property `write.merge.mode` to "merge-on-read".
```

Comment:
Do we need to add that format-version needs to be v2?

Also nit: I think 'need to' is redundant, removing it seems more consistent with the rest of docs